spec/workers/pull_request_monitor/pr_branch_record_spec.rb

```diff
@@ -0,0 +1,106 @@
+require "spec_helper"
+
+RSpec.describe PullRequestMonitor::PrBranchRecord do
+  describe ".create" do
+    it "has git create a local pr branch" do
+      repo = spy("CommitMonitorRepo")
```

Comment:
My only thought was that spies are really for after-the-fact expectation checks, so it seems weird to use them as_null_object, but it is clever :smile:  I'm fine with it for now.